index.html

```diff
@@ -0,0 +1,137 @@
+            2021
+          </p>
+
+          <div class="text-container">
+            <p class="welcome-text">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Laborum
+              minus, porro, tenetur rem voluptatum suscipit quisquam dolor
+              ullam.
+            </p>
+          </div>
+
+          <div class="date">2021.12.25(THU) ~ 16(FRI)</div>
+          <div class="location">@ Tech Event Center, Wuse 2 Abuja</div>
+        </div>
+      </div>
+
+      <section class="programs">
+        <h2>Main Program</h2>
+        <div class="programs-container">
+          <div class="program">
+            <div class="program-title">
+              <span><img src="./images/png/path1183.png" alt="" /></span
+              ><span>Forum</span>
+            </div>
+            <div class="program-description">
+              Listen to the speakers from various countries about the messeges
+              of sharing and opening
+            </div>
+          </div>
+
+          <div class="program">
+            <div class="program-title">
+              <span><img src="./images/png/path1183.png" alt="" /></span
+              ><span>Exhibition</span>
+            </div>
+            <div class="program-description">
+              Listen to the speakers from various countries about the messeges
+              of sharing and opening
+            </div>
+          </div>
+
+          <div class="program">
+            <div class="program-title">
+              <span><img src="./images/png/Vector1.png" alt="" /></span
+              ><span>Lecture</span>
+            </div>
+            <div class="program-description">
+              Listen to the speakers from various countries about the messeges
+              of sharing and opening
+            </div>
+          </div>
+
+          <div class="program">
+            <div class="program-title">
+              <span><img src="./images/png/Vector.png" alt="" /></span
+              ><span>Chops</span>
+            </div>
+            <div class="program-description">
+              Listen to the speakers from various countries about the messeges
+              of sharing and opening
+            </div>
+          </div>
+
+          <div class="program">
+            <div class="program-title">
+              <span><img src="./images/png/Vector.png" alt="" /></span
+              ><span>Chops</span>
+            </div>
+            <div class="program-description">
+              Listen to the speakers from various countries about the messeges
+              of sharing and opening
+            </div>
+          </div>
+        </div>
+        <div class="btn btn-program">Join AR DEV Summit 2021</div>
+      </section>
+
+      <section class="speakers">
+        <h2>Featured Speakers</h2>
+
+        <div class="speaker-container"></div>
+
+        <div class="btn btn-speaker">More <i class="fa fa-angle-down"></i></div>
+      </section>
+    </main>
+
```

Comment:
- [ ] You have forgotten the footer in your **home page (index.html)**, please add the footer as you have in the **about page**.

- your design

![fffff](https://user-images.githubusercontent.com/61562626/142161718-3c839ec4-bd4d-405a-b668-2f1d3888296e.jpeg)



- given design
![Capture web_17-11-2021_9639_www behance net](https://user-images.githubusercontent.com/61562626/142161598-1a4a55ab-4e59-4763-8d78-e90b7fd3716f.jpeg)